pagegraph-cli/src/request_id_info.rs

```diff
@@ -24,6 +24,7 @@ pub fn main(graph: &PageGraph, request_id_arg: usize, frame_id: Option<FrameId>)
         // RequestComplete
         resource_type: String,
         status: String,
+        source: Option<String>,
         value: Option<String>,
```

Comment:
My vote would be on just populating the `value` field if we don't already have it from the `RequestComplete` edge, rather than creating a new field for it. This seems like a workaround for something missing from the generated graphs, not a distinct piece of information (unless you've found something else from your investigation @pes10k)